notebooks/Connecting_with_Python_(GDS_client).ipynb

```diff
@@ -51,76 +42,64 @@
         "    auth=(AURA_USERNAME, AURA_PASSWORD),\n",
         "    aura_ds=True\n",
         ")"
-      ],
-      "metadata": {
-        "id": "o1shsW0Jv9Dw"
-      },
-      "execution_count": null,
-      "outputs": []
+      ]
     },
     {
       "cell_type": "markdown",
-      "source": [
-        "In this example the `gds.version` procedure can be used to retrieve the version of GDS running on the database."
-      ],
       "metadata": {
         "id": "htdxTDMmv5ba"
-      }
+      },
+      "source": [
+        "In this example the `gds.version` procedure can be used to retrieve the version of GDS running on the database."
```

Comment:
does this need to say instance rather than database?

Reply:
Good one, haven't even noticed it